worlds/sc2wol/Locations.py

```diff
@@ -258,15 +271,19 @@ def get_locations(world: Optional[MultiWorld], player: Optional[int]) -> Tuple[L
                      lambda state: state._sc2wol_has_competent_comp(world, player)),
         LocationData("Shatter the Sky", "Shatter the Sky: Leviathan", SC2WOL_LOC_ID_OFFSET + 2805,
                      lambda state: state._sc2wol_has_competent_comp(world, player)),
-        LocationData("All-In", "All-In: Victory", None)
+        LocationData("All-In", "All-In: Victory", None,
+                     lambda state: state._sc2wol_final_mission_requirements(world, player))
     ]
 
     beat_events = []
 
-    for location_data in location_table:
-        if location_data.name.endswith((": Victory", ": Defeat")):
+    for i in range(len(location_table)):
```

Comment:
`for i, location_data in enumerate(location_table)` to avoid retrieving location data several times via index

Reply:
Did not know about that function.  Fixed and pushed.